frontend/src/app/pages/settings-page/settings-page.component.html

```diff
@@ -16,18 +16,22 @@
                     </mat-expansion-panel-header>
 
                     <mat-form-field>
-                        <input matInput placeholder="First name" type="text" [formControl]="userName" required readonly>
+                        <input matInput placeholder="First name" type="text" [formControl]="userName" required>
```

Comment:
I think that formControl should have the same name as the field: so it would be `[formControl]="userFirstName" `